vast/CMakeLists.txt

```diff
@@ -21,22 +29,14 @@ macro (create_tenzir_symlink name)
     CODE "
     execute_process(COMMAND
       \"${CMAKE_COMMAND}\" -E create_symlink \"./tenzir\"
-      \"\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${name}\")"
+      \"\$ENV{DESTDIR}\${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_BINDIR}/${name}\")"
```

Comment:
This does not look correct to me, shouldn't DESTDIR from ENV affect the install prefix already?

Reply:
No it doesn't. Look at the `cmake_install.cmake` file in `build/vast/` and you will see.

Reply:
If that's the case then all the other instances of `install(CODE …` should be adapted as well.